migrations/versions/0084_add_created_to_job_stats.py

```diff
@@ -0,0 +1,22 @@
+"""empty message
+
+Revision ID: 0084_add_created_to_job_stats
+Revises: 0081_add_job_stats
+Create Date: 2017-05-11 15:05:53.420946
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '0084_add_created_to_job_stats'
+down_revision = '0083_add_job_stats'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.add_column('job_statistics', sa.Column('created_at', sa.DateTime(), nullable=True))
```

Comment:
it was added as later in the story as needed for the delete task. At the time I was wondering whether it should be a separate PR so it stayed a separate script. Didn't think it mattered too much.